### PR TITLE
docs(color): update hex code for red accent

### PR DESCRIPTION
**Pull request by glynnis**
JIRA: https://jira.rax.io/browse/FRMW-978

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM by @ssalinas24 

# Before

screen shot 2016-07-26 at 11 31 22 am
screen shot 2016-07-26 at 11 31 17 am

# After
[Outdated screenshot.](https://cloud.githubusercontent.com/assets/1529366/17146683/e21d173e-5324-11e6-8396-4672a9106efd.png)

screen shot 2016-07-26 at 1 10 45 pm
screen shot 2016-07-26 at 11 31 31 am

@Droogans Guessing this doesn't need a QE LGTM but feel free to add back a checkbox if you think it's called for.

### CODE REVIEW

File: demo/templates/styles/color.html [hunk 1]
```diff
@@ -314,13 +314,31 @@
...
+            Used for link colors and icons associated with destructive/destroy actions.
+            Irreversible changes (like rebuilding a server) are not considered destructive actions.
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="palette red">
+      <div class="color accent light-text">
+        <div>Red - Error/Destructive (dark background)<br />700</div>
```
Comment (ssalinas24) [inline demo/templates/styles/color.html]: Remove `700` number. Perhaps replace with `@red-accent` i assume? You agree?

**Comment by CITguy** [comment]
LGTM - functional. I'll defer to @ssalinas24 for design and copy updates.

File: demo/templates/styles/color.html [hunk 2]
```diff
@@ -314,13 +314,31 @@
...
+        <div>Red - Error/Destructive (light background)<br />700</div>
         <div>@red-700 <small>#d32f2f</small></div>
       </div>
       <div class="detail">
         <ul class="list">
           <li>
-            Used to mean "error", either as part of a response or status message.
+            Used to mean "error", either as part of a response or status message on a light background.
+          </li>
+          <li>
```
Comment (ssalinas24) [inline demo/templates/styles/color.html]: I think we should add an `li` for "Used for required field asterisks and invalid input borders on a light background".

File: demo/templates/styles/color.html [hunk 3]
```diff
@@ -314,13 +314,31 @@
...
+      <div class="color accent light-text">
+        <div>Red - Error/Destructive (dark background)<br />700</div>
+        <div>@red-accent <small>#ec515f</small></div>
+      </div>
+      <div class="detail">
+        <ul class="list">
+          <li>
+            Used to mean "error", either as part of a response or status message on a dark background.
           </li>
           <li>
```
Comment (ssalinas24) [inline demo/templates/styles/color.html]: I think we should add an `li` for "Used for required field asterisks and invalid input borders on a dark background".

**Comment by ssalinas24** [comment]
@glynnis I have reviewed and added my feedback. Glad you were able to get to this so quick. Thank you.

**Comment by glynnis** [comment]
@ssalinas24 Made changes, updated screenshots.

File: demo/templates/styles/color.html (hunk 1 above)
Comment (glynnis) [inline demo/templates/styles/color.html]: I put just `Accent` to match the way the accent swatches appear at the top of the page.

**Comment by ssalinas24** [comment]
Design LGTM

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**